inst/tinytest/test_d_delayed_5.R

```diff
@@ -25,7 +25,7 @@ library(tiledbcloud)
 
 # ----------------------------------------------------------------
 a <- delayed(function() { 9 }, name='a', local=FALSE)
-expect_error(compute(a, timeout_seconds=300),
+expect_error(compute(a, namespace=NULL< timeout_seconds=300),
```

Comment:
Is the '<' a typo here? Did you mean to delete 'timeout_seconds=300'?

Reply:
Fixed typo, thanks @eddelbuettel ! :)